.github/workflows/test_objectivec.yml

```diff
@@ -68,17 +68,27 @@ jobs:
     strategy:
       fail-fast: false   # Don't cancel all jobs if one fails.
       matrix:
-        PLATFORM: ["ios", "macos", "tvos", "watchos"]
+        PLATFORM: ["ios", "macos", "tvos", "watchos", "visionos"]
         CONFIGURATION: ["Debug", "Release"]
-    name: CocoaPods ${{ matrix.PLATFORM}} ${{ matrix.CONFIGURATION}}
-    runs-on: macos-12
+        include:
+          - OS: macos-12
+          - XCODE: Xcode_14.1
+          - PLATFORM: "visionos"
+            OS: macos-14
+            XCODE: Xcode_15.2
+    name: CocoaPods ${{ matrix.PLATFORM }} ${{ matrix.CONFIGURATION }}
+    runs-on: ${{ matrix.OS }}
     env:
-      DEVELOPER_DIR: /Applications/Xcode_14.1.app/Contents/Developer
+      DEVELOPER_DIR: /Applications/${{ matrix.XCODE }}.app/Contents/Developer
     steps:
       - name: Checkout pending changes
         uses: protocolbuffers/protobuf-ci/checkout@v2
         with:
           ref: ${{ inputs.safe-checkout }}
+      - name: Update cocoapods
```

Comment:
Why is this step needed, the images already have cocoapods on them.

Reply:
The version of cocoapods is 1.14.3 before https://github.com/actions/runner-images/pull/9377 get into merge from yesterday, I will delete this step.